Dockerfile

```diff
@@ -9,11 +9,11 @@ WORKDIR /app
 # Install dependencies
 RUN apt-get update && apt-get install -y jq
 
-RUN git clone https://github.com/MainframeHQ/go-ethereum.git -b swarm-network-rewrite
+RUN git clone https://github.com/ethereum/go-ethereum.git
```

Comment:
Shouldn't we use the `ethesphere` org here? Or is the main `go-ethereum` regularly in sync?

Reply:
The commit we need to fix the tar download behavior is only in the ethereum repo right now, that's why I used it: https://github.com/ethereum/go-ethereum/commit/3bcb501c8fefeec1bc8aac26686460cae86f2ccb